mern/server/server.js

```diff
@@ -1,9 +1,9 @@
-import express from 'express';
-import cors from 'cors';
-import mongoose from 'mongoose'
-import dotenv from 'dotenv'
```

Comment:
Just to clarify, both single and double quotations work in js right?

Reply:
Yes, I just have prettier set up so it replaces single quotes with double, so when I save it automatically changes it.

Reply:
Oh that's super useful!